hapi-fees/src/main/java/com/hedera/services/usage/token/TokenOpsUsage.java

```diff
@@ -51,6 +61,10 @@
 	private static final int ROYALTY_HBAR_FALLBACK_REPR_SIZE = ROYALTY_NO_FALLBACK_REPR_SIZE + FIXED_HBAR_REPR_SIZE;
 	private static final int ROYALTY_HTS_FALLBACK_REPR_SIZE = ROYALTY_NO_FALLBACK_REPR_SIZE + FIXED_HTS_REPR_SIZE;
 	private static final long LONG_BASIC_ENTITY_ID_SIZE = BASIC_ENTITY_ID_SIZE;
+	private static final int AMOUNT_REPR_BYTES = 8;
```

Comment:
is this long size ? I see that this is used just for `expiry`. If so we can remove this and just use `LONG_SIZE`

Reply:
Yeah, replaced.